ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml

```diff
@@ -35,6 +35,26 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ipi-aws
+- as: e2e-upgrade-functests
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-upgrade-functests
+      cli: latest
+      commands: |
+        make deploy-kubevirt # Should deploy Kubevirt and CDI and wait for their CRs to report 'Available'
+        make deploy-kubevirt-ssp-operator # Should deploy kubevirt-ssp-operator (https://github.com/kubevirt/kubevirt-ssp-operator) and wait for its 4 CRs to report 'Available'
```

Comment:
These could be shell scripts in the `hack` directory, or `automation` directory, not make targets.

Reply:
Yea that's better, will change